tests/test_zcl.py

```diff
@@ -388,12 +388,28 @@ async def test_write_wrong_attribute(cluster):
         assert cluster._write_attributes.call_count == 1
 
 
+async def test_write_unknown_attribute(cluster):
+    with patch.object(cluster, "_write_attributes", new=AsyncMock()):
+        # Using an invalid attribute name, the call should fail
+        res = await cluster.write_attributes({"dummy_attribute": 5})
+        assert isinstance(res.exception(), KeyError)
```

Comment:
```
with pytest.raises(KeyError):
    await cluster.write_attributes({"dummy_attribute": 5})
```

Reply:
Thanks, this worked (tried another method as well that didn't).